Source/WebCore/layout/formattingContexts/inline/InlineTextItem.cpp

```diff
@@ -89,7 +89,7 @@ bool InlineTextItem::isZeroWidthSpaceSeparator() const
 
 bool InlineTextItem::isQuirkNonBreakingSpace() const
 {
-    if (style().nbspMode() != NBSPMode::Space || style().textWrap() == TextWrap::NoWrap || style().whiteSpaceCollapse() == WhiteSpaceCollapse::BreakSpaces)
+    if (style().nbspMode() != NBSPMode::Space || style().textWrapMode() == TextWrapMode::NoWrap || style().whiteSpaceCollapse() == WhiteSpaceCollapse::BreakSpaces)
```

Comment:
I really wish we had a boolean for `text-wrap-mode` instead of an enumeration for call sites like this.

Reply:
Same here